pkgs/servers/timetagger/default.nix

```diff
@@ -24,8 +24,6 @@ python3.pkgs.buildPythonApplication {
     echo "#!${python3.interpreter}" >> $out/bin/timetagger
     cat timetagger/__main__.py >> $out/bin/timetagger
     chmod +x $out/bin/timetagger
-    wrapProgram $out/bin/timetagger \
-      --set TIMETAGGER_BIND "${addr}:${toString port}"
```

Comment:
This is a breaking change and I do not think it is a good idea.

If you really want to do this, at least also remove the `, addr ? "127.0.0.1"` `, port ? 8082` at the top of this file.